src/App.js

```diff
@@ -3,29 +3,35 @@ import ReactDOM from "react-dom/client";
 import avatar from '../assets/avatar.png';
 import cards from "./data/cards.json";
 import CardComponent from "./components/CardComponent";
+import SearchBar from "./components/SearchBar";
+import { useState } from "react";
 
 const LogoComponent = () => <img className="logo" src="https://lwfiles.mycourse.app/635a5db526fa77c557bab00d-public/c4377b05799f40987f16cacd43da014d.png"></img>;
-const SearchComponent = () => <input type="text" placeholder="Search Team Member.."></input>;
 const UserAvatarComponent = () => <img className="avatar" src={avatar}></img>;
 
 const HeadingComponent = () => (
   <div id="header" className="header" tabIndex="1">
     <span><LogoComponent /></span>
-    <span className="search-form"><SearchComponent/></span>
     <span><UserAvatarComponent/></span>
 
   </div>
 );
-const CardContainer = () =>
- cards.map((teammate) => (
+const CardContainer = ({ filtertedTeammates }) =>
+filtertedTeammates.map((teammate) => (
```

Comment:
Can you change variable naming to camel case ? I think this is only at one place so may be you missed out by mistake.